internal/pkg/service/buffer/store/model/task.go

```diff
@@ -12,7 +13,7 @@ type Task struct {
 	CreatedAt  UTCTime        `json:"createdAt" validate:"required"`
 	FinishedAt *UTCTime       `json:"finishedAt,omitempty"`
 	Node       string         `json:"node"` // validate:"required"`
-	Lock       string         `json:"lock" validate:"required"`
+	Lock       etcdop.Key     `json:"lock" validate:"required"`
```

Comment:
Ulozime si rovno dany typ (odvodeny od `string`), aby sa to nemuselo v kode pretypovavat.